awx/main/notifications/slack_backend.py

```diff
@@ -16,13 +16,17 @@
 class SlackBackend(AWXBaseEmailBackend):
 
     init_parameters = {"token": {"label": "Token", "type": "password"},
+                       "hex_color": {"label": "Notification Color", "type": "string"},
                        "channels": {"label": "Destination Channels", "type": "list"}}
     recipient_parameter = "channels"
     sender_parameter = None
 
-    def __init__(self, token, fail_silently=False, **kwargs):
+    def __init__(self, token, hex_color, fail_silently=False, **kwargs):
```

Comment:
@paihu thanks for the contribution!

Does this upgrade reliably?  I'm wondering how this would work for notifications that are set up in awx versions _prior_ to this change (which don't define a `hex_color`).  Perhaps we could make this argument _optional_, e.g.,?

```python
def __init__(self, token, hex_color='', fail_silently=False, **kwargs):
```